common/lib/common-utils/src/performanceNode.ts

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { performance as nodePerformance } from "perf_hooks";
+import { IsomorphicPerformance } from "./performanceIsomorphic";
+
+export const performance: IsomorphicPerformance = nodePerformance;
```

Comment:
Typing to `IsomorphicPerformance` to prevent node typings from leaking to dependent packages.